pages/common/prosodyctl.md

```diff
@@ -0,0 +1,23 @@
+# prosodyctl
+
+> The control tool for the Prosody XMPP server.
+
+- Show the status of the Prosody server:
+
+`sudo prosodyctl status`
+
+- Reload the server's configuration files:
+
+`sudo prosodyctl reload`
+
+- Add a user to the Prosody XMPP server:
+
+`sudo prosodyctl adduser user@example.com`
+
+- Set a user's password:
+
+`sudo prosodyctl passwd user@example.com`
+
+- Permanently delete a user:
```

Comment:
Is there anything like temporarily delete a user ? If no, then its simpler as "Delete a user"